cfgov/v1/models/base.py

```diff
@@ -539,3 +539,9 @@ class TemporaryLockout(models.Model):
     user = models.ForeignKey(User)
     created = models.DateTimeField(auto_now_add=True)
     expires_at = models.DateTimeField()
+
+
+class Feedback(models.Model):
+    is_helpful = models.BooleanField()
+    comment = models.TextField(blank=True, null=True)
+    page = models.ForeignKey(Page, on_delete=models.SET_NULL, null=True)
```

Comment:
Using `models.SET_NULL` here seems better than [`CASCADE`](https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.ForeignKey.on_delete), but still means that if someone sends feedback for a page, and that page is then deleted, then we'll have feedback left in the DB with no way to associate it to a page.

Could you just use the page URL here, instead of a foreign key? That'd record the equivalent information and would be simpler, especially if this feedback is just intended for export to some other format.

If you do keep this as a foreign key, can you add a more useful [`related_name`](https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.ForeignKey.related_name) here, so that you could refer to e.g. `Page.feedback`?